lib/abide_dev_utils/xccdf.rb

```diff
@@ -218,6 +229,20 @@ def controls_by_profile_title(profile_title)
         profiles.select { |x| x.title == profile_title }.controls
       end
 
+      def gen_map(dir: nil, type: 'CIS', parent_key_prefix: '', console: false)
```

Comment:
If you don't use `console`, you can just add `**_` as the last argument in this method signature. What that means is it stores all other keyword args that aren't explicitly stated in an unused variable.

Reply:
Neat!